LogMonitor/src/LogMonitor/LogFileMonitor.cpp

```diff
@@ -810,7 +811,7 @@ LogFileMonitor::LogFilesChangeHandler()
     const DWORD eventsCount = 3;
 
     LARGE_INTEGER liDueTime;
-    INT64 millisecondsToWait = 30000LL;
+    INT64 millisecondsToWait = 1000LL;

```

Comment:
Where is ` m_waitInSeconds` used? Does it have any relationship with this?

Reply:
Also consider providing a brief comment about the rationale behind the 1sec delay.

Reply:
The m_waitInSeconds is not rerated to the 30 seconds delay encountered (the comment introduced was a small omission in the initial  implementation). It's related to the liDueTime introduced by SetWaitableTimer.

Another fix option considered was to remove all the liDueTime usage to set waitable timer in LogFileMonitor.cpp, however I have noted some performance regression introduced by this method.

Reply:
I see, thanks!